src/test/java/edu/cornell/kfs/concur/batch/fixture/PdpFeedDetailEntryFixture.java

```diff
@@ -0,0 +1,55 @@
+package edu.cornell.kfs.concur.batch.fixture;
+
+import java.util.EnumSet;
+
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.concur.ConcurTestConstants.PdpFeedFileConstants;
+import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedDetailEntry;
+
+public enum PdpFeedDetailEntryFixture {
+    MARSHAL_TEST( 105.28, EnumSet.of(PdpFeedAccountingEntryFixture.TRANSACTION_AMOUNT_25_50, PdpFeedAccountingEntryFixture.TRANSACTION_AMOUNT_71_45)),
+    DETAIL_ONE_TRANS_ZERO(0, EnumSet.of(PdpFeedAccountingEntryFixture.TRANSACTION_AMOUNT_0)),
+    DETAIL_ONE_TRANS_ZERO_ONE_TRANS_POSITIVE(25.50, EnumSet.of(PdpFeedAccountingEntryFixture.TRANSACTION_AMOUNT_0, PdpFeedAccountingEntryFixture.TRANSACTION_AMOUNT_25_50)),
+    DETAIL_TWO_TRANS_SUM_TO_POSITIVE(61.45, EnumSet.of(PdpFeedAccountingEntryFixture.TRANSACTION_AMOUNT_71_45, PdpFeedAccountingEntryFixture.TRANSACTION_AMOUNT_NEGATIVE_10)),
+    DETAIL_TWO_TRANS_SUM_TO_ZERO(0, EnumSet.of(PdpFeedAccountingEntryFixture.TRANSACTION_AMOUNT_10, PdpFeedAccountingEntryFixture.TRANSACTION_AMOUNT_NEGATIVE_10)),
+    DETAIL_TWO_TRANS_SUM_TO_NEGATIVE(-5, EnumSet.of(PdpFeedAccountingEntryFixture.TRANSACTION_AMOUNT_5, PdpFeedAccountingEntryFixture.TRANSACTION_AMOUNT_NEGATIVE_10));
+    
+    public final String sourceDocNbr;
+    public final String invoiceNbr;
+    public final String invoiceDate;
+    public final double origInvoiceAmt;
+    public final String fsOriginCd;
+    public final String fdocTypCd;
+    public final EnumSet<PdpFeedAccountingEntryFixture> accountingEntries;
```

Comment:
The constructor line sounds correct, but for the field on the enum, you could use Set instead of Collection if you want to.